### ISSUE TITLE
Add term retracted tympanic membrane

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10351]
**Preferred term label:**
Retracted tympanic membrane

**Synonyms**
Retracted eardrum, Tympanic membrane retraction

**Definition (free text, please give PubMed ID)**
A condition in which part of the tympanic membrane is retracted into the middle ear cavity (also called retraction pocket)
PMID: 32890292

**Parent term (use hpo.jax.org/app)**
Abnormal tympanic membrane morphology HP:0040090

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
PMID: 32890292

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw 
Adding new term: 
- ID: HP:6000459
- label: Retracted tympanic membrane
- parents: [Abnormal tympanic membrane morphology (HP:0040090)](https://hpo.jax.org/app/browse/term/HP:0040090)
- pmid: PMID:32890292
- def: A condition in which part of the tympanic membrane is pull backed into the middle ear cavity (also called retraction pocket).
- comment: 
- orcid: orcid:0009-0007-1020-5412
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10351
- synonyms:
